### PR TITLE
Added WorkingDirectory from issue #8418

### PR DESCRIPTION
**Pull request by tchentchen**
Had this been in the docs, but would've saved me two hours. Saving someone else two hours. :)

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_docs/autostart/systemd.markdown [hunk 1]
```diff
@@ -31,6 +31,7 @@ Description=Home Assistant
 After=network.target
 
 [Service]
+WorkingDirectory=/home/homeassistant
```
Comment (fabaff) [inline source/_docs/autostart/systemd.markdown]: If systemd run as user this shouldn't be needed as it points to the user's (`User=`) home directory as far as I remember.

Comment (frenck) [reply source/_docs/autostart/systemd.markdown]: Would it hurt? I guess not...

**Comment by frenck** [comment]
@fabaff @tchentchen 

This PR is now open for a month. Anything we can do to move it forward?

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Pull request closed without merging**